css/style.css

```diff
@@ -0,0 +1,51 @@
+.graybackground{
+  background-color: rgb(234, 229, 229);
+}
+main{
+  display: block;
+  float:right;
+}
+#main-doc{
+  line-height: 150%;
+  position: absolute;
+  margin-left: 310px;
+}
```

Comment:
Please make sure to have one space between the name of the class or Id and the subsequent opening curly brace so as to pass linter checkers.